readthedocs.yml

```diff
@@ -2,4 +2,4 @@ build:
     image: latest
 
 python:
-    version: 3.6
+    version: 3.7
```

Comment:
Should this actually be Python 3.7? I don't know much about how this file is used

Reply:
This is the version of the Python interpreter RTD uses when building the sphinx docs. It's probably fine either way. Since Py3.7 should be backwards compatible, I'm in favor of leaving it because we always seem to forget to update the RTD files whenever we make other changes.